SalesforceCommerceCloud/pwa-kit
# Release `1.1.2` - Snyk security upgrades and fix `ansi-regex` vulnerability [release-1.1.x]

GUS: https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE000008ur0JYAQ/view

# Description
This PR releases version `1.1.2` with these security upgrades:
- Snyk security upgrade `validator`. This was done in the original Snyk PR: https://github.com/SalesforceCommerceCloud/pwa-kit/pull/189
- Fix `ansi-regex` vulnerability:
  - Upgrade `copy-webpack-plugin` to latest. This upgrade was already done on the `develop` branch https://github.com/SalesforceCommerceCloud/pwa-kit/pull/181. Here we apply the same upgrade to the `release-1.1.x` branch.
  - Upgrade `webpack-notifier` to latest. Fixing the `ansi-regex@5.0.0` vulnerability found in `webpack-notifier` transient dependencies.


# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Upgrade `copy-webpack-plugin`, `validator` and `webpack-notifier` dependencies.

# How to Test-Drive This PR

1. Checkout this branch `snyk-20211110-1.1.x`.
2. Inside the `projects/pwa-kit-react-sdk` folder run `npm ls ansi-regex`
3. Verify that all versions should be `5.0.1` (safe version to use, upgraded from 5.0.0) with the exception of 2 usages of `2.2.1` which is not vulnerable.
4. Run `snyk test --strict-out-of-sync=false --all-projects` at the root of the project and verify that there are no new high vulnerabilities. 

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
